Sources/ChatManager.swift

```diff
@@ -50,7 +50,7 @@ import NotificationCenter
 
         self.logger = logger
 
-        let sdkInfo = PPSDKInfo(productName: "chatkit", sdkVersion: "1.6.0")
+        let sdkInfo = PPSDKInfo(productName: "chatkit", sdkVersion: "1.7.0")
```

Comment:
sucks that you have to manually update this every time. I wonder if you can pull it out of the podspec? (out of scope for this PR, but would be cool if it isn't a lot of work)